src/session/content/item_row.rs

```diff
@@ -94,16 +96,290 @@ impl ItemRow {
+                                                            // Translators: This comma is used to separete names of two users
+                                                            first_members.join(&gettext(", ")),
+                                                            last_member
+                                                        ))
+                                                    }
+                                                )
+                                            }
+                                        }
+                                    }
+                                ));
+                            }
+                            MessageContent::MessageChatDeleteMember(data) => {
+                                self.get_or_create_event_row().set_label(&format!(
+                                    "<b>{}</b>",
+                                    match message.sender() {
+                                        MessageSender::User(user) if user.id() == data.user_id => {
+                                            if message.is_outgoing() {
+                                                gettext("You left the group")
+                                            } else {
+                                                gettext!("{} left the group", sender_name)
+                                            }
+                                        }
+                                        _ => {
+                                            let user_name = self::sender_name(
+                                                &message
+                                                    .chat()
+                                                    .session()
+                                                    .user_list()
+                                                    .get(data.user_id),
+                                            );
+                                            if message.is_outgoing() {
+                                                gettext!("You removed {} from the group", user_name)
+                                            } else {
+                                                gettext!(
+                                                    "{} removed {} from the group",
+                                                    sender_name,
+                                                    user_name
+                                                )
+                                            }
+                                        }
+                                    }
+                                ));
                             }
-                            _ => self.set_child_row::<MessageText>(message),
+                            MessageContent::MessagePinMessage(data) => {
+                                let msg =
+                                    match message.chat().history().message_by_id(data.message_id) {
```

Comment:
This line still causes occasionally crashes #229 might fix it though. 
I've got this error, it's to be investigated in the future
`thread 'main' panicked at 'already mutably borrowed: BorrowError', src/session/chat/history.rs:154:25`